packages/core-components/api-report.md

```diff
@@ -515,6 +516,13 @@ export function MarkdownContent(props: Props_11): JSX.Element;
 // @public (undocumented)
 export function MissingAnnotationEmptyState(props: Props_6): JSX.Element;
 
+// Warning: (ae-missing-release-tag) "MobileSidebar" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const MobileSidebar: ({
```

Comment:
It's public because it stays in a different file than the Sidebar and is therefor exposed. Is there another way to document this (is `@internal` meant for that?)

Reply:
Yeah you can put an `@internal` on it if you like